app/utils/url_utils.rb

```diff
@@ -43,4 +43,13 @@ def extract_locale_from_url(url)
   def strip_port_from_host(host)
     host.split(":").first
   end
+
+  # Naive join method, which can be used to normalize multiple slashes
+  #
+  # Usage: URLUtils.join("foo", "bar/", "baz") => "foo/bar/baz"
+  def join(*parts)
+    parts.select(&:present?).reduce { |a, b|
+      "#{a.sub(/\/+\z/, "")}/#{b}"
```

Comment:
Yeah, I thought about the same, but found this: https://www.ruby-forum.com/topic/50137. To make sure, perhaps the `File.join` call can be wrapped in setting and restoring the `File::SEPARATOR`?